src/yunohost/tools.py

```diff
@@ -306,8 +306,6 @@ def tools_postinstall(operation_logger, domain, password, ignore_dyndns=False,
     if 'redirected_urls' not in ssowat_conf:
         ssowat_conf['redirected_urls'] = {}
```

Comment:
Imho we can also remove this `if`

(I guess keeping the other stuff that read/write the conf.json.persistent doesn't hurt so that at least the file exists already, so people willing to tweak it won't have to create it manually)